docs/user/automation-rules.rst

```diff
@@ -1,96 +1,110 @@
-Automation Rules
+Automation rules
```

Comment:
haha just read this again and realized that it sounds a lot like "automation rules [verb]"

Maybe that'll make people click it, anyways :)